rfc-drafts/stream.md

```diff
@@ -221,31 +231,142 @@ existing code, perhaps as part of an edition migration.
 
 Designing such a migration feature is out of scope for this RFC.
 
-## IntoStream / FromStream traits, mirroring iterators
+## IntoStream / FromStream traits
+
+The `IntoStream` trait would provide a way to convert something into a `Stream`.
```

Comment:
Probably good to elaborate a bit here and to mention that: Iterators have an `IntoIterator` trait that is used with `for` loops. We might want such a trait for streams, and it would look like ...